.kokoro/nightly/tutorials-samples.cfg

```diff
@@ -0,0 +1,33 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Do we need a separate build for tutorials, as opposed to just samples?

Reply:
We with Neenu have discussed the approach, she suggested the tests to be run with the Nightly Job. So I have added this config and new JOB_TYPE to run just tutorial-samples.

Is that fine or you would like to implement some other approach?

Reply:
There is a ["nightly" samples](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-retail%2Fnightly%2Fsamples?search_pattern=cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-retail%2F&project_types=KOKORO&include_inactive_projects=false) build already happening. Do we need a separate job for the tutorials, or can we just keep them all in one job?

Reply:
The reasoning was to keep interactive tutorials separate from the regular samples for ease of code maintenance. Unless there is a strong recommendation against it, let's keep it separate.